CHANGELOG

```diff
@@ -1,3 +1,10 @@
+Version 0.3.1 (2019-09-19)
+--------------------------
+Update scala to 2.11.1 and sbt to 0.13.18
+Update referers.yml to latest version from https://s3-eu-west-1.amazonaws.com/snowplow-hosted-assets/third-party/referer-parser/referers-latest.yml
+Read referrer config file (DB) from S3 containing latest list of referrers (https://s3-eu-west-1.amazonaws.com/snowplow-hosted-assets/third-party/referer-parser/referers-latest.yml) with fallback to referrers hardcoded in referer.yml
```

Comment:
I think it might be better to only update the hardcoded file. The ability to update from S3 is nice and will be very useful, but my feeling is that we should keep this patch as minimal as possible.

We'll depend on 0.3.1 in `Snowplow R117` and there is no way to know how long people will stay on that release for -- so this patched version of the library might end up with a much longer lifetime that we want.

Reply:
If the functionality is there, why not use it. It should not make the release harder to maintain if R118 gets delayed. And if R118 takes a long time to arrive it will be very convenient that our customers are always able to use a recent version of the database.

Reply:
I was thinking in terms of how we integrate this version with the rest.

We have currently 0.5.0 on `master` and a PR for 0.6.0. 

0.3.1 branches off 0.3.0 and leaves us with two options:
- rebase 0.4.0 on top of 0.3.1, which might result in cascading conflicts all the way up to 0.6.0
- leave 0.4.0 to be based on 0.3.0, in which case 0.3.1 becomes this weird thing to the side that nobody remembers in 1 year.

If we end up with the second case, I think this creates problems for open source users, because 0.3.1 will be basically ignored and buried; and yet it will be a dependency in R117, so anyone on that release might need to look for it.

If we end up with the first case, I think the rebasing will be much easier, if we only changed a line here and there and did not add major new changes.